### PR TITLE
Styling/form labels

### LINKED ISSUE #96: Labels for Fly when editing or adding
Add labels in the Fly editing and adding page so that it is clearer of what the user is entering when setting fly information.

### PR DESCRIPTION
**Pull request by rachael-t**
### Issue(s) related to this PR:
* closes #96
 
 
 ### Type of change made:
* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [X] Styling- no new features

#### If this submission is a new feature:
* [X] Have you added an explanation of what your changes do?
* [X] Have you lint your code locally prior to submission?

### Change Log:
* Added labels to the FlyForm 

### Were there any challenges that arose? If so, how were they addressed?
* No, but we did realize that there is a bug needing to be addressed (#100)

### Technical Debt:
* None

### Testing:
* [X] Have you written new tests for your core changes, as applicable?
* [X] Have you successfully ran tests with your changes locally?
* [X] All new and existing tests passed?
* [X] Have you run linter?

**Test coverage** = 45.93%

### CODE REVIEW

**Comment by C-Bandstra** [comment]
Thank you for getting this done so fast!!

File: ApiCalls.js [hunk 1]
```diff
@@ -136,3 +136,25 @@ export const deleteFish = async (fishId) => {
     console.error(error);
   }
 };
+
+// Image upload for species identification:
+export const uploadPhoto = async (photo) => {
+  try {
+    const response = await fetch(
+      "https://my-fly-box-api.herokuapp.com/api/v1/images",
```
Comment (TimNguyen21) [inline ApiCalls.js:144]: A good call to move this to the ApiCalls.  This make it the proper formatting for the fetch calls.

**Review by C-Bandstra — APPROVED** [review]

File: components/FlyForm.js [hunk 2]
```diff
@@ -19,25 +19,28 @@ class FlyForm extends Component {
   handleChange = (property, text) => {
     this.setState({ property: text });
     this.props.updateFlyEntry(property, text);
-  }
+  };
 
   render() {
     return (
       <View style={styles.container}>
+        <Text style={styles.label}>Fly Name:</Text>
```
Comment (TimNguyen21) [inline components/FlyForm.js:27]: Thanks for adding these labels for these input fields in the fly editing and adding page.  This make things more understandable for the user when filling out fly information.

Comment (rachael-t) [reply components/FlyForm.js:27]: You're welcome! Thanks for bringing this to the group's attention!

**Review by TimNguyen21 — APPROVED** [review]
Thanks for adding the labels in the Fly form.  This is a clean way to show the user what fields they are using.

**Pull request merged**